docs/concepts.md

```diff
@@ -0,0 +1,28 @@
+---
+id: concepts
+title: Temporal concepts
+description: The following concepts have specific definitions within the context of the Temporal Platform.
+sidebar_label: Concepts
+tags:
+  - introduction
+---
+
+The [Temporal Platform](/temporal) provides developers a suite of effective tools for building reliable applications at scale.
+
+The concepts, components, and features of the platform are described in detail across the concept guides.
+
+- [Temporal](/temporal)
+- [Workflows](/workflows)
+- [Activities](/activities)
+- [Retry Policies](/retry-policies)
+- [Tasks](/tasks)
+- [Workers](/workers)
+- [Clusters](/clusters)
+- [Namespaces](/namespaces)
+- [Visibility](/visibility)
```

Comment:
The sequence of terms in this list differs from the current version of the sidebar. Which one should rule?